packages/battery/android/build.gradle

```diff
@@ -32,29 +32,3 @@ android {
         disable 'InvalidPackage'
     }
 }
-
-// TODO(amirh): Remove this hack once androidx.lifecycle is included on stable. https://github.com/flutter/flutter/issues/42348
-afterEvaluate {
```

Comment:
🎉

Reply:
Should this wait a little longer though to wait for the traffic to ramp up?

Reply:
right. Let's not merge this for now. It was just an exercise for us to look at what the new code would look like.

Reply:
raised the min sdk version to 1.12. Merging on green.